webpack.config.js

```diff
@@ -98,6 +96,8 @@ const babelLoader = {
 const webpackConfig = {
 	bail: ! isDevelopment,
 	entry: {},
```

Comment:
entry is the same for prod and dev, might as well inline it here

Reply:
is it? I thought entry was a weird array in dev. We overwrite it though, so ... could inline prod.